src/Processor/Braintree/Mapper/MapsModifiers.php

```diff
@@ -18,8 +18,16 @@ protected function fromGenericResult(ModifierBuilder $builder, $result): void
             $builder->withQuantity($result->quantity);
         }
 
+        // Braintree blows up when you access $result->propertyThatIsNotSet, so check for the value to be set first.
+        $neverExpires = $result->__isset('neverExpires') ? $result->neverExpires : true;
```

Comment:
Regular `isset()` does not work here? Because `isset($result->neverExpires)` triggers the error mentioned? Calling the magic method looks odd.

Reply:
@ismyrnow and I fought this for way longer than it should... they are overriding the magic method on their `Base` class to check a private `$_attributes` value...

https://github.com/braintree/braintree_php/blob/master/lib/Braintree/Base.php#L52-L62

Reply:
you were right... I thought we had tried that and got bad results... but maybe I made a mistke. It's happened once or twice before.